TLSharp.Core/TelegramClient.cs

```diff
@@ -258,11 +258,22 @@ public async Task<Boolean> SendTypingAsync(TLAbsInputPeer peer)
             return await SendRequestAsync<Boolean>(req);
         }
 
-        public async Task<TLAbsDialogs> GetUserDialogsAsync()
+        public async Task<TLAbsDialogs> GetUserDialogsAsync(int offset_date = 0, int offset_id = 0, TLAbsInputPeer offset_peer = null, int limit = 100)
         {
-            var peer = new TLInputPeerSelf();
-            return await SendRequestAsync<TLAbsDialogs>(
-                new TLRequestGetDialogs() { OffsetDate = 0, OffsetPeer = peer, Limit = 100 });
+            if (!IsUserAuthorized())
+                throw new InvalidOperationException("Authorize user first!");
+
+            if (offset_peer == null)
```

Comment:
we use camelCase notation, not under_score, for parameter names

Reply:
FYI: Please look at original code line 316. it was already using 'max_id' under_score parameter names.